### PR TITLE
Add sidebar in editform-constructor

### PR DESCRIPTION
**Pull request by KollegOFF**
TFS: [#163422](http://dis-tfs:8080/Common/CASEBERRY/_workitems/edit/163422).

### CODE REVIEW

File: addon/controllers/fd-editform-constructor.js [hunk 1]
```diff
@@ -1,11 +1,71 @@
...
 export default Ember.Controller.extend({
   queryParams: ['classId'],
 
+  /**
+    @private
+    @property _selecetedIsControl
+    @readOnly
+    @type Boolean
+  */
+  _selecetedIsControl: Ember.computed('selectedControl', function() {
```
Comment (savolkov) [inline addon/controllers/fd-editform-constructor.js]: Is it a typo or a feature? `_selectedIsControl`

File: addon/controllers/fd-editform-constructor.js [hunk 2]
```diff
@@ -1,11 +1,71 @@
...
+    return this.get('selectedControl') instanceof FdEditformControl;
+  }).readOnly(),
+
+  /**
+    @private
+    @property _selecetedIsGroup
+    @readOnly
+    @type Boolean
+  */
+  _selecetedIsGroup: Ember.computed('selectedControl', function() {
```
Comment (savolkov) [inline addon/controllers/fd-editform-constructor.js]: Same

File: addon/controllers/fd-editform-constructor.js [hunk 3]
```diff
@@ -1,11 +1,71 @@
...
+    return this.get('selectedControl') instanceof FdEditformGroup;
+  }).readOnly(),
+
+  /**
+    @private
+    @property _selecetedIsTab
+    @readOnly
+    @type Boolean
+  */
+  _selecetedIsTab: Ember.computed('selectedControl', function() {
```
Comment (savolkov) [inline addon/controllers/fd-editform-constructor.js]: Same

File: app/templates/fd-editform-constructor/form-config-panel.hbs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          {{t "forms.fd-editform-constructor.form-config-panel.control-properties-header"}}
+        </div>
+        {{#if _selecetedIsControl}}
+          <div class="field">
+            <label>{{t "forms.fd-editform-constructor.form-config-panel.control-caption"}}</label>
+            {{flexberry-textbox value=selectedControl.caption}}
+          </div>
+          <div class="field">
+            <label>{{t "forms.fd-editform-constructor.form-config-panel.control-type"}}</label>
+            {{flexberry-dropdown items=(array "int" "bool" "string" "date") value=selectedControl.type}}
```
Comment (savolkov) [inline app/templates/fd-editform-constructor/form-config-panel.hbs]: I'm not really sure, but maybe it is better to create array in controller?

Comment (KollegOFF) [reply app/templates/fd-editform-constructor/form-config-panel.hbs]: Yes. I do not know yet what it will be, this is for example.

**Pull request merged**